src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public abstract class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stop working");
```

Comment:
The stop message 'Excavator stop working' is grammatically incorrect and could be more informative. Consider changing it to 'Excavator has stopped working' to maintain consistency with the start message and to improve readability.